torch/cuda/nvtx.py

```diff
@@ -0,0 +1,46 @@
+import os
+import ctypes
+import warnings
+import torch.cuda
```

Comment:
Remove the unused imports. It would also be good to import this from `torch/cuda/__init__.py`. That way the `torch.cuda.nvtx.xxx` functions are immediately available when `torch` is imported.